bin/install_protoc.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Could this also take into account changes in istio/tools/protoc-doc-gen? When that changes, we also need to run go generate.

Reply:
It should already do that..

Whenever a user runs make depend.diff, it will run go generate ./...
Also, looks like circle ci always runs depend.diff, so it would run go generate ./... too all the time unless steps above it fails..